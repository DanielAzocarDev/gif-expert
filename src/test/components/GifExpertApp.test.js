import { shallow } from "enzyme"
import GifExpertApp from '../../GifExpertApp'

describe('test <GifExpertApp/>', () => {

  

  test('should render component', () => {
    const wrapper = shallow(<GifExpertApp/>)

    expect(wrapper).toMatchSnapshot()
  })

  test('should have 2 categories', () => {

    const categories = ['goku', 'naruto']

    const wrapper = shallow(<GifExpertApp defaultCategories={categories}/>)

    expect(wrapper.find('GifGrid').length).toBe(categories.length)
  })
})