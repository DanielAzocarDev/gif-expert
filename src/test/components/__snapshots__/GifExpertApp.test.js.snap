// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <GifExpertApp/> should render component 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
